persistence-typed/src/main/scala/org/apache/pekko/persistence/typed/javadsl/EventSourcedBehavior.scala

```diff
@@ -16,6 +16,8 @@ package org.apache.pekko.persistence.typed.javadsl
 import java.util.Collections
 import java.util.Optional
 
+import scala.jdk.OptionConverters._
```

Comment:
there is a Pekko OptionConverters - can you use that? org.pekko.util

We can't use scala.jdk.OptionConverters because we still support Scala 2.12